packages/react-store/tests/index.test.tsx

```
@@ -1,4 +1,11 @@
-import { act, render, renderHook, waitFor } from '@testing-library/react'
+import { Suspense, startTransition, useState, use } from 'react'
```

ESLint

[error] Member 'use' of the import declaration should be sorted alphabetically.

(sort-imports)